package.json

```diff
@@ -21,61 +35,42 @@
     }
   ],
   "files": [
-    "dist/",
-    "src/",
-    "postinstall.js"
+    "src/"
```

Comment:
You really want to limit the entry points to the module to prevent misuse.  You don't want people importing from cross-domain-utils/src because they don't know what the underlying code is and how it needs to be compiled.  Like they would need that transform for the for/in loop etc.  You don't want to allow import from cross-domain-utils/dist because things may change such as file names. 

For webpack/rollupjs when you import it is going to go to package.json -> module property -> /dist/module/**/*.  This is babel compiled source code with ES6 modules so tree shaking will work.  This adheres to the webpack recommendation.

"module will point to a module that has ES2015 module syntax but otherwise only syntax features that browser/node supports."

https://webpack.js.org/guides/author-libraries/

For everything else it will go to package.json -> main property -> dist/bundle/cross-domain-utils.js which is the webpack bundle that will work in browsers.

I don't think minification is really an issue because when you import from cross-domain-utils you will get /dist/module/**/* which is the compiled source.  The library that is using CDU as a dependency would do the minification which would minify it all.

Once all the libraries are not importing from src/ anymore we need to remove the files array entirely so that main and module are the only two entry points.  This will prevent misuse of importing and prevent headaches in the future.